src/components/editor/DraftSpec.tsx

```diff
@@ -60,6 +73,25 @@ function DraftSpecEditor() {
         }
     }, [currentCatalog]);
 
+    useEffectOnce(() => {
```

Comment:
Listen for new changes from the server. This is WAY too simple but okay for right now.